### DISCUSSION TITLE
2022/02/17/%E3%80%8A%E6%AF%9B%E9%80%89%E3%80%8B%EF%BC%9A%E6%98%9F%E6%98%9F%E4%B9%8B%E7%81%AB%EF%BC%8C%E5%8F%AF%E4%BB%A5%E7%87%8E%E5%8E%9F/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#15] [Announcements]
# 2022/02/17/%E3%80%8A%E6%AF%9B%E9%80%89%E3%80%8B%EF%BC%9A%E6%98%9F%E6%98%9F%E4%B9%8B%E7%81%AB%EF%BC%8C%E5%8F%AF%E4%BB%A5%E7%87%8E%E5%8E%9F/

在对于时局的估量和伴随而来的我们的行动问题上，我们党内有一部分同志还缺少正确的认识。他们虽然相信革命高潮不可避免地要到来，却不相信革命高潮有迅速到来的可能。因此他们不赞成争取江西的计划，而只赞成在福建、广东、江西之间的三个边界区域的流动游击，同时也没有在游击区域建立红色政权的深刻的观念，因此也就没有用这种红色政权的巩固和扩大去促进全国革命高潮的深刻的观念。他们似乎认为在距离革命高潮尚远的时期做这种

http://localhost:4000/2022/02/17/%E3%80%8A%E6%AF%9B%E9%80%89%E3%80%8B%EF%BC%9A%E6%98%9F%E6%98%9F%E4%B9%8B%E7%81%AB%EF%BC%8C%E5%8F%AF%E4%BB%A5%E7%87%8E%E5%8E%9F/#fnref:18

**Comment by Westlifers** [comment]
本文是为了测试脚注插件，目前还有点问题。脚注序号会自动重复且相互重叠
